cmd/snap/cmd_connect_test.go

```diff
@@ -62,24 +62,31 @@ Help Options:
 
 func (s *SnapSuite) TestConnectExplicitEverything(c *C) {
 	s.RedirectClientToTestServer(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
for now, yes. At some point we'll get the websocket working, and then it'll be: POST, connect to websocket, wait on it